pkg/unikontainers/unikontainers.go

```diff
@@ -427,18 +433,23 @@ func (u *Unikontainer) saveContainerState() error {
 	return os.WriteFile(stateName, data, 0o644) //nolint: gosec
 }
 
-// ExecuteHooks executes concurrently any hooks found in spec based on name:
 func (u *Unikontainer) ExecuteHooks(name string) error {
+	return u.executeHooksConcurrently(name)
+	// return u.executeHooksSequentially(name)
```

Comment:
Let's add a comment here to explain how to choose the hook execution mode. We can also add it in our future configuration options.

Reply:
I refactored this to be more "configuration friendly". I also added a comment explaining how to toggle between the two modes.